manifests/server/database.pp

```diff
@@ -85,7 +89,16 @@
       $database_suffix = ''
     }
 
-    $subname = "//${database_host}:${database_port}/${database_name}${database_suffix}"
+    $subname_default = "//${database_host}:${database_port}/${database_name}${database_suffix}"
```

Comment:
The default `database_host` is `localhost`, but for the certificate to validate I needed to make it be the `fqdn`.

One tricky thing is that the default binding of the postgresql service is to 127.0.0.1, but for the FQDN to resolve it needs to be bound to `0.0.0.0`. As a hack while I was testing I added an entry to `/etc/hosts` but I'm not sure that's a viable solution for the module.